CBLClient/PredictiveQueries.py

```diff
@@ -0,0 +1,68 @@
+from CBLClient.Args import Args
+from CBLClient.Client import Client
+
+
+class PredictiveQueries(object):
+    _db = None
+    _baseUrl = None
+
+    def __init__(self, base_url):
+        self.base_url = base_url
+
+        # If no base url was specified, raise an exception
+        if not self.base_url:
+            raise Exception("No base_url specified")
+
+        self._client = Client(base_url)
+
+    def registerModel(self, modelName):
+        args = Args()
+        args.setString("model_name", modelName)
+        return self._client.invokeMethod("predictiveQuery_registerModel", args)
+
+    def unregisterModel(self, modelName):
+        args = Args()
+        args.setString("model_name", modelName)
+        return self._client.invokeMethod("predictiveQuery_unRegisterModel", args)
+
+    def getPredictionQueryResult(self, model, dictionary, database):
+        args = Args()
+        args.setMemoryPointer("model", model)
+        args.setMemoryPointer("dictionary", dictionary)
+        args.setMemoryPointer("database", database)
+        return self._client.invokeMethod("predictiveQuery_getPredictionQueryResult", args)
+
+    def queryNonDictionaryInput(self, model, dictionary, database):
+        args = Args()
+        args.setMemoryPointer("model", model)
+        args.setString("dictionary", dictionary)
```

Comment:
I like this method name. :) 
so for non dictionary query, shall use other var name than dictionary? dictionary is a string here, right?

Reply:
changed the name to relevant name 'nonDictionary"